src/incentives/README.md

```diff
@@ -1,4 +1,4 @@
-# Incentives API
+# Incentives aPI
```

Comment:
@Psalmz777 was this small change intended? Seems to be relatively strange? :)

Reply:
Yes it was..I used that to check if the token was working ,i have changed it back